### ISSUE TITLE
Whitelabel error page

### ISSUE THREAD

**Issue by thijskh** [OpenConext/OpenConext-voot#1]
https://voot.example.org/aaa shows:
```
Whitelabel Error Page

This application has no explicit mapping for /error, so you are seeing this as a fallback.
```

A simple error template similar to OpenConext-authorization-server would be fine.

**Comment by oharsta** [comment]
Fixed in 644d8f6a363dbb38b3cb69da6b3d0d5a0f718c57 by adding custom JSON error controller

**Comment by oharsta** [comment]
As this is a JSON end-point the output for errors is also json:

```
{ 
timestamp: 1441618490484,
status: 404,
error: "Not Found",
message: "No message available",
path: "/sdfds"
}
```
